lib/server/index.js

```diff
@@ -44,3 +58,8 @@ module.exports = ({ config = {}, port = null, filters = {} }) => {
     },
   };
 };
+
+
+function hash(s) {
+  return 'sha512::' + createHash('sha512').update(s).digest('hex');
```

Comment:
Isn't creating a hash object somewhat computationally heavy? Why not create one on startup and use it?